2-mandatory/2-financial-times.js

```diff
@@ -14,24 +17,43 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let fewestWordTitle = allArticleTitles[0];
+    for (const article of allArticleTitles) {
+        if (article.length < fewestWordTitle.length) {
+            fewestWordTitle = article;
+        }
+    }
+    return fewestWordTitle;
 }
 
 /*
     The editor of the FT has realised that headlines which have numbers in them get more clicks!
     Implement the function below to return a new array containing all the headlines which contain a number.
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
-function headlinesWithNumbers(allArticleTitles) {
-    // TODO
-}
+
+
+ function headlinesWithNumbers(allArticleTitles) {
+    let articleWithNums = [];
+    for (let i = 0; i < allArticleTitles.length; i++) {
```

Comment:
Similarly you already used `forEach` for other solutions, feel free to do that here as well